core/src/main/java/org/apache/gravitino/storage/relational/mapper/ModelVersionAliasSQLProviderFactory.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.gravitino.storage.relational.mapper;
+
+import com.google.common.collect.ImmutableMap;
+import java.util.List;
+import java.util.Map;
+import org.apache.gravitino.storage.relational.JDBCBackend.JDBCBackendType;
+import org.apache.gravitino.storage.relational.mapper.provider.base.ModelVersionAliasRelBaseSQLProvider;
+import org.apache.gravitino.storage.relational.mapper.provider.postgresql.ModelVersionAliasRelPostgreSQLProvider;
+import org.apache.gravitino.storage.relational.po.ModelVersionAliasRelPO;
+import org.apache.gravitino.storage.relational.session.SqlSessionFactoryHelper;
+import org.apache.ibatis.annotations.Param;
+
+public class ModelVersionAliasSQLProviderFactory {
+
+  static class ModelVersionAliasRelMySQLProvider extends ModelVersionAliasRelBaseSQLProvider {}
+
+  static class ModelVersionAliasRelH2Provider extends ModelVersionAliasRelBaseSQLProvider {}
+
+  private static final Map<JDBCBackendType, ModelVersionAliasRelBaseSQLProvider>
+      MODEL_VERSION_META_SQL_PROVIDER_MAP =
+          ImmutableMap.of(
+              JDBCBackendType.MYSQL, new ModelVersionAliasRelMySQLProvider(),
+              JDBCBackendType.H2, new ModelVersionAliasRelH2Provider(),
+              JDBCBackendType.POSTGRESQL, new ModelVersionAliasRelPostgreSQLProvider());
+
+  public static ModelVersionAliasRelBaseSQLProvider getProvider() {
+    String databaseId =
+        SqlSessionFactoryHelper.getInstance()
+            .getSqlSessionFactory()
+            .getConfiguration()
+            .getDatabaseId();
+
+    JDBCBackendType jdbcBackendType = JDBCBackendType.fromString(databaseId);
+    return MODEL_VERSION_META_SQL_PROVIDER_MAP.get(jdbcBackendType);
+  }
+
+  public static String insertModelVersionAliasRel(
```

Comment:
Should make the `Rel` to `Rels` for the both method name and the param? Because we need insert a list here.

Reply:
OK, I will update the code.